pkg/addons/seaweedfs/seaweedfs.go

```diff
@@ -0,0 +1,208 @@
+package seaweedfs
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/k0sproject/k0s/pkg/apis/k0s/v1beta1"
+	"github.com/replicatedhq/embedded-cluster/pkg/helpers"
+	"github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"gopkg.in/yaml.v2"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/replicatedhq/embedded-cluster/pkg/kubeutils"
+	"github.com/replicatedhq/embedded-cluster/pkg/spinner"
+)
+
+const (
+	releaseName = "seaweedfs"
+	namespace   = "seaweedfs"
+)
+
+// Overwritten by -ldflags in Makefile
+var (
+	ChartURL  = "https://url"
+	ChartName = "name"
+	Version   = "v0.0.0"
+)
+
+var helmValues = map[string]interface{}{
+	"master": map[string]interface{}{
+		"replicas": 1,
+		"data": map[string]interface{}{
+			"type": "persistentVolumeClaim",
+			"size": "1Gi",
+		},
+		"logs": map[string]interface{}{
+			"type": "persistentVolumeClaim",
+			"size": "1Gi",
+		},
+		"disableHttp": true,
+	},
+	"volume": map[string]interface{}{
+		"replicas": 1,
+		"dataDirs": []map[string]interface{}{
+			{
+				"name":         "data1",
+				"type":         "persistentVolumeClaim",
+				"size":         "10Gi",
+				"storageClass": "openebs-hostpath",
+				"maxVolumes":   0,
```

Comment:
Default `maxVolumes` is `7` which would limit how much data can be stored even if the underlying disk has space left. Let's set it to a high number so we don't have to worry about it in the future:
```suggestion
				"maxVolumes":   100,
```
That would roughly allow up to `100 * 30GB = 3TB` disk space to be consumed by the registry bucket. I don't think vendors would ever need more than that, right?

Reply:
No reason not to make it even a bigger number like `1000` 🤷‍♂️. Since we want it to be able to consume whatever storage is available.